tests/test_utils.py

```diff
@@ -856,6 +856,27 @@ def test_csv_dialect_detection_semicolon_delimited(self):
         self.assertEqual(lookup_dict[4]['header3'], 'bar')
         self.assertEqual(lookup_dict[1]['header1'], 1)
 
+    def test_csv_utf8_encoding(self):
+        """utils: test that CSV read correctly with UTF-8 encoding."""
+        from natcap.invest import utils
+
+        csv_file = os.path.join(self.workspace_dir, 'csv.csv')
+        # writing with utf-8-sig will prepend the BOM
```

Comment:
We want to remove this comment, right?

Reply:
Sorry, I thought I'd removed these already. Lost that change somehow I guess